praise-participants.md

```diff
@@ -0,0 +1,60 @@
+## Title
+Praise Participants
+
+## Patlet
+After an inner source contribution it's important to thank the contributor for their time and effort.
+This pattern gives guidance for a way to do so that not only effectively acknowledges the contribution but also endgenders further engagement from the contributor and others.
+
+## Problem
+How can we properly express our gratitude to a contributor for their inner source contribution to a project?
+It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity.
+A pattern in this area makes it easy to do and ensures that the message comes across clearly and sincerely.
+
+## Context
+* You are the trusted committer or maintainer on an inner source project.
+* You value the community of contributors and want to maintain and grow it.
+
+## Forces
+You are busy, which makes it easy to forget some of the soft touches like praise and thanks.
+You may not be someone that is comfortable in social situations or good with words.
```

Comment:
An additional force: The praise has a different effect depending from where it comes from. In my experience, praise coming from someone who is recognized as a leader in the field is more valuable to many engineers than from s.o. who has no such _street creds_. TCs should be such a leader, but it might still be helpful to mention that.

Reply:
Another idea, based on a study done by IBM way back when: performance only helps with having a career to a very small degree (10 %). More important is _image_, and praise will contribute to that.

Reply:
`Peer recognition is very important to job satisfaction and career development.`